packages/wine.rb

```diff
@@ -33,8 +33,8 @@ def self.build
   def self.install
     case ARCH
     when "x86_64"
-      system "mkdir -p #{CREW_DEST_DIR}/usr/local/bin"
-      FileUtils.cd("#{CREW_DEST_DIR}/usr/local/bin") do
+      system "mkdir -p #{CREW_DEST_PREFIX}/bin"
```

Comment:
`FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"`?

Reply:
You know, at this point is it about unhardcoding paths or making little unneeded fixes?